tokio/src/sync/mpsc/mod.rs

```diff
@@ -70,13 +70,12 @@
 //!
 //! # Multiple runtimes
 //!
-//! The mpsc channel does not care about which runtime you use it in, and can be
-//! used to send messages from one runtime to another. It can also be used in
-//! non-Tokio runtimes.
-//!
-//! There is one exception to the above: the [`send_timeout`] must be used from
-//! within a Tokio runtime, however it is still not tied to one specific Tokio
-//! runtime, and the sender may be moved from one Tokio runtime to another.
+//! The mpsc channel is runtime agnostic and can be used to send messages
+//! between different runtimes, including non-Tokio runtimes.
+//! The only exception being [`send_timeout`], which needs to be called from
+//! a Tokio runtime.
+//! For more details see
+//! [the documentation of the `sync` module](crate::sync#runtime-compatibility).
```

Comment:
Perhaps we could just copy the text from the `tokio::sync` docs verbatim? (of course, with "synchronization primtivie" replaced with "mpsc channel".)

Reply:
That works for me. Should we change the title of the section from `Multiple runtimes` to `Runtime compatibility` as well then? I was a bit hesitant to do so because it would break links to the section on the `latest` release on docs.rs.

Reply:
That seems like a good reason to keep the current title.